Mathlib/Topology/ContinuousMap/CompactlySupported.lean

```diff
@@ -641,6 +641,83 @@ noncomputable def nnrealPart (f : C_c(α, ℝ)) : C_c(α, ℝ≥0) where
 lemma nnrealPart_apply (f : C_c(α, ℝ)) (x : α) :
     f.nnrealPart x = Real.toNNReal (f x) := rfl
 
+/-- The compactly supported continuous `ℝ≥0`-valued function as a compactly supported `ℝ`-valued
+function. -/
+noncomputable def toReal (f : C_c(α, ℝ≥0)) : C_c(α, ℝ) :=
+  f.compLeft ContinuousMap.coeNNRealReal
+
+@[simp]
+lemma toReal_apply (f : C_c(α, ℝ≥0)) (x : α) : f.toReal x = f x := compLeft_apply rfl
+
+lemma eq_nnrealPart_neg_nnrealPart (f : C_c(α, ℝ)) :
+    f = (nnrealPart f).toReal - (nnrealPart (-f)).toReal := by
+  ext x
+  simp
+
+/-- The compactly supported continuous `ℝ≥0`-valued function as a compactly supported `ℝ`-valued
+function. -/
+noncomputable def toRealLinearMap : C_c(α, ℝ≥0) →ₗ[ℝ≥0] C_c(α, ℝ) where
+  toFun f := f.compLeft ContinuousMap.coeNNRealReal
+  map_add' f g := by
+    ext x
+    simp
+  map_smul' a f := by
+    ext x
+    simp [NNReal.smul_def]
+
+@[simp]
+lemma LinearMap.toReal_apply (f : C_c(α, ℝ≥0)) (x : α) :
+    LinearMap.toReal f x = (f x).toReal := rfl
+
+lemma LinearMap.coe_toReal (f : C_c(α, ℝ≥0)) :
+    LinearMap.toReal f = f.toReal := by
+  ext x
+  simp
+
+/-- For a positive linear functional `Λ : C_c(α, ℝ) → ℝ`, define a `ℝ≥0`-linear map. -/
+noncomputable def toNNRealLinear {Λ : C_c(α, ℝ) →ₗ[ℝ] ℝ} (hΛ : ∀ f, 0 ≤ f → 0 ≤ Λ f) :
```

Comment:
```suggestion
noncomputable def toNNRealLinear (Λ : C_c(α, ℝ) →ₗ[ℝ] ℝ) (hΛ : ∀ f, 0 ≤ f → 0 ≤ Λ f) :
```
to avoid proofs being elided in the infoview